src/components/ProductListItem/style.css

```diff
@@ -0,0 +1,80 @@
+.ProductListItem {
```

Comment:
You can remove most of the classes by using the utility classes provided by tailwindcss.

Reply:
We can do this together if you want to.